### PR TITLE
fix: keyboard navigation and mouse hover

### LINKED ISSUE #12: Keyboard Navigation Broken
I just realized now that the keyboard navigation is not loading the new previews. Before you could use the up and down arrow keys to surf through the issues quickly.

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves #12

- keyboard navigation is working again using up and down arrows
- when hovering with the mouse, disabled the effect set from keyboard navigation

Should consider having some unit tests afterwards 😄

### CODE REVIEW

**Comment by 0x4007** [comment]
> Should consider having some unit tests afterwards 😄

If you like writing tests I can keep you busy. They are desperately needed but everybody hates doing them in this environment when we are iterating rapidly. 

Sadly for me I seemed to have broken the continuous deployment last week and I don't have my MacBook to run this locally. Will have to check this evening.

**Comment by 0x4007** [comment]
The code looks fine. @rndquu I'm not sure if you're familiar with this feature but perhaps you can get around to checking and merging this in before me tonight.

**Comment by gentlementlegen** [comment]
> > Should consider having some unit tests afterwards 😄
> 
> If you like writing tests I can keep you busy. They are desperately needed but everybody hates doing them in this environment when we are iterating rapidly.
> 
> Sadly for me I seemed to have broken the continuous deployment last week and I don't have my MacBook to run this locally. Will have to check this evening.

I am not exception! But I believe it's always beneficial in the long run. Feel free to create Issues for testing, since I set up one already we have sort of a base testing config that we can reuse so it might be quicker.

**Review by rndquu — APPROVED** [review]
Keyboard top and down arrows change focused issues fine

**Comment by 0x4007** [comment]
> > > Should consider having some unit tests afterwards 😄
> 
> > 
> 
> > If you like writing tests I can keep you busy. They are desperately needed but everybody hates doing them in this environment when we are iterating rapidly.
> 
> > 
> 
> > Sadly for me I seemed to have broken the continuous deployment last week and I don't have my MacBook to run this locally. Will have to check this evening.
> 
> 
> 
> I am not exception! But I believe it's always beneficial in the long run. Feel free to create Issues for testing, since I set up one already we have sort of a base testing config that we can reuse so it might be quicker.

In our DAO anybody is welcome to make new proposals by filing new issues in the appropriate repositories. The bot has direct financial incentives for making detailed specifications (if they are funded)

A core team member will come in and set a time and priority label to fund it. 

The next step is to have somebody (or the original issue author) to submit a deliverable that will close the GitHub issue. 

Basically there are direct financial incentives I encourage you to take advantage of.
